packages/themes/src/tokens/color.styles.tokens.json

```diff
@@ -0,0 +1,23 @@
+{
+  "Skeleton-gradient": {
+    "$type": "gradient",
+    "$value": {
+      "type": "linear",
+      "angle": 90,
+      "stops": [
+        {
+          "color": "#dedede00",
+          "position": 0
+        },
+        {
+          "color": "#ffffff29",
+          "position": 0.55
+        },
+        {
+          "color": "#dedede00",
+          "position": 1
+        }
+      ]
+    }
+  }
+}
```

Comment:
nitpicking: All tokens JSON files are missing an empty line at the end of the file. Should we add the empty line?